core-plugins/src/main/java/co/cask/hydrator/plugin/batch/aggregator/function/CollectList.java

```diff
@@ -0,0 +1,44 @@
+package co.cask.hydrator.plugin.batch.aggregator.function;
```

Comment:
We require a copyright for all files, please add the same copyright from the other files, except with 2019 as the year.

Reply:
Done